src/foundry/foundry.js/config.d.ts

```diff
@@ -1531,16 +1535,27 @@ declare global {
         /** @defaultValue `{ layerClass: ControlsLayer, group: "interface" }` */
         controls: LayerDefinition<ConstructorOf<ControlsLayer>>;
 
-        [key: string]: LayerDefinition<ConstructorOf<CanvasLayer>>;
+        [key: string]: LayerDefinition;
+      }
+
+      interface CanvasGroup extends PIXI.Container {
+        sortableChildren: boolean;
+      }
+
+      interface CanvasGroupConstructor extends ConstructorOf<PIXI.Container> {
```

Comment:
`ConstructorOf` is not the right thing to extend here because it looses all of the static properties that `typeof PIXI.Container` has. Unfortunately, you cannot use `typeof` in `extends`, so we need to define it as a type first:
```ts
type PixiContainerConstructor = typeof PIXI.Container;
interface CanvasGroupConstructor extends PixiContainerConstructor { ... }
```
we can just declare that type at module scope, so that it isn't available globally.